native/cocos/renderer/pipeline/custom/LayoutGraphJsb.cpp

```diff
@@ -29,6 +29,7 @@
  * ========================= !DO NOT CHANGE THE FOLLOWING SECTION MANUALLY! =========================
  */
 // clang-format off
+#include "cocos/bindings/auto/jsb_gfx_auto.h"
```

Comment:
Added missing gfx jsb header. Otherwise, there will be multiple jsb template specialization, which will result in multiple definition in c++ linking stage.